ephios/core/signup/structure/base.py

```diff
@@ -94,12 +94,9 @@ def get_shift_state_context_data(self, request, **kwargs):
         Additionally to the context of the event detail view, provide context for rendering `shift_state_template_name`.
         """
         kwargs["shift"] = self.shift
-        kwargs["participations"] = self.shift.participations.filter(
-            state__in={
-                AbstractParticipation.States.REQUESTED,
-                AbstractParticipation.States.CONFIRMED,
-            }
-        ).order_by("-state")
+        kwargs["participations"] = sorted(
```

Comment:
doesn't that render declined participations for the other methods?

Reply:
yes yes I'll get to that :sweat_smile: